redbot/core/core_commands.py

```diff
@@ -1428,7 +1429,7 @@ async def regionalformat(self, ctx: commands.Context, language_code: str = None)
     @_set.command()
     @checks.is_owner()
     async def custominfo(self, ctx: commands.Context, *, text: str = None):
-        """Customizes a section of [p]info
+        """Customizes a section of [p]info.
```

Comment:
Quotation needed, but I believe we decided that command names in help docstrings in Red should be put in backquotes. (don't change this until someone confirms)
```suggestion
        """Customizes a section of `[p]info`.
```

Reply:
You know... Just do it 👍